public/assets/sass/components/_delete.scss

```diff
@@ -0,0 +1,91 @@
+#deleteCard {
```

Comment:
IMHO (In My Humble Opinion) this could have a better generic name like "modal" component.